packages/content/src/content.ts

```diff
@@ -76,11 +171,13 @@ function enrichContent(content: IHubContent, options?: IGetContentOptions) {
   if (shouldFetchOrgId(content)) {
     propertiesToFetch.orgId = getOwnerOrgId;
   }
-  if (Object.keys(propertiesToFetch).length === 0) {
-    // nothing more to fetch
-    return content;
-  }
-  return _fetchContentProperties(propertiesToFetch, content, options);
+
+  const fetchContentPropertiesPromise: Promise<IHubContent> =
+    Object.keys(propertiesToFetch).length === 0
+      ? Promise.resolve(content)
+      : _fetchContentProperties(propertiesToFetch, content, options);
+
+  return fetchContentPropertiesPromise.then(_enrichDates);
```

Comment:
this fn does not have a return type... not sure how the lint/ts compiler let that slide... 🤷‍♂️

Reply:
Not sure I understand your comment @dbouwman

Reply:
He means we had not specified a return type (but I have now on line 183 with `: Promise<IHubContent>`).

Reply:
Ah. Yes, TypeScript won't complain about that unless it can't infer the return type through static code analysis. In this case, it can since you've clearly defined a return type for `_enrichDates` and the compiler sees that your promise will resolve to that return type.

Reply:
Not saying it's a bad idea to add the explicit return type to this function. Just explaining the answer to Dave's comment
> not sure how the lint/ts compiler let that slide